CONTRIBUTING.rst

```diff
@@ -1,4 +1,72 @@
-Contributing to ``pycodestyle``
+Contributing to pycodestyle
 ===============================
 
-Please see the `developer notes <https://pycodestyle.readthedocs.io/en/latest/developer.html>`_
+When contributing to pycodestyle, please observe our `Code of Conduct`_.
+
+Step 1: Forking pycodestyle for editing
+---------------------------------------
+
+Fork the pycodestyle repository on github. This will add
+pycodestyle to your github account. You will push your changes to your
+fork and then make pull requests into the official pycodestyle repository.
+
+Github has an excellent `guide`_ that has screenshots on how to do this.
+
+Next, clone your fork of the pycodestyle repository to your system for
```

Comment:
Was this meant to be Step 2 ? If not, you just need to renumber steps 3 and 4 below to 2 and 3.